planetarium/NineChronicles
# chore: bump lib9c into 6d141945ef87417d4414476e6f67c43014c17ff6

### Description

1. It bumps Lib9c into https://github.com/planetarium/lib9c/commit/6d141945ef87417d4414476e6f67c43014c17ff6.

### How to test

1. Check the build success on your local because the CircleCI build seems broken 😢 

### Related Links

- https://github.com/planetarium/lib9c/commit/6d141945ef87417d4414476e6f67c43014c17ff6.

### Required Reviewers

@planetarium/9c-dev 

### Screenshot

None.
